### PR TITLE
Add proc_access_win_invoke_patchingapi

### PR DESCRIPTION
**Pull request by frack113**
eg: https://github.com/D1rkMtr/UnhookingPatch
```
Process accessed:
RuleName: -
UtcTime: 2023-01-07 09:27:53.806
SourceProcessGUID: {8b8a3bea-3b19-63b9-d702-00000000bd00}
SourceProcessId: 8380
SourceThreadId: 5696
SourceImage: C:\git\UnhookingPatch\UnhookingPatch\PatchingAPI\x64\Debug\PatchingAPI.exe
TargetProcessGUID: {8b8a3bea-3b19-63b9-d802-00000000bd00}
TargetProcessId: 6676
TargetImage: C:\Windows\SYSTEM32\calc.exe
GrantedAccess: 0x1FFFFF
CallTrace: C:\Windows\SYSTEM32\ntdll.dll+9e9b4|C:\Windows\System32\KERNELBASE.dll+9053|C:\Windows\System32\KERNELBASE.dll+789e|C:\Windows\System32\KERNELBASE.dll+7446|C:\Windows\System32\KERNEL32.DLL+678ec|UNKNOWN(000001F92C7500E4)
SourceUser: SIMU\Administrateur
TargetUser: SIMU\Administrateur
```

### CODE REVIEW

File: rules/windows/process_access/proc_access_win_invoke_patchingapi.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - attack.t1562.002
+logsource:
+    category: process_access
+    product: windows
+detection:
+    selection:
+        GrantedAccess: '0x1FFFFF'
+        CallTrace|startswith: 'C:\Windows\SYSTEM32\ntdll.dll+'
+        CallTrace|contains: '|UNKNOWN('
+    condition: selection
```
Comment (nasbench) [inline rules/windows/process_access/proc_access_win_invoke_patchingapi.yml:21]: The rule 5f113a8f-8b61-41ca-b90f-d374fa7e4a39 has a similar case.

Comment (frack113) [reply rules/windows/process_access/proc_access_win_invoke_patchingapi.yml:21]: 5f113a8f-8b61-41ca-b90f-d374fa7e4a39  is a `1 of selection* ` so get many FP. 
This rule need all
I should add  ` CallTrace|endswith: ')'`

File: rules/windows/process_access/proc_access_win_invoke_patchingapi.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+title: Suspect PatchingAPI
```
Comment (nasbench) [inline rules/windows/process_access/proc_access_win_invoke_patchingapi.yml:1]:
```suggestion
title: Potential NT API Stub Patching
```

File: rules/windows/process_access/proc_access_win_invoke_patchingapi.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+id: b916cba1-b38a-42da-9223-17114d846fd6
+status: experimental
+description: Detects suspect access  such as that used by PatchingAPI
```
Comment (nasbench) [inline rules/windows/process_access/proc_access_win_invoke_patchingapi.yml:4]:
```suggestion
description: Detects potential NT API stub patching as seen used by the project PatchingAPI
```

**Review by nasbench — APPROVED** [review]
Just change the author to yourself.

**Pull request merged**